test/callback.html

```diff
@@ -0,0 +1,81 @@
+
+      <!DOCTYPE html>
+      <html lang="en">
+        <head>
+          <meta charset="utf-8" />
+          <meta name="viewport" content="width=device-width, initial-scale=1" />
+          <title>trevor</title>
+          <style> 
+            body {
+                margin: 0;
+                padding: 0;
+            }
+          </style>
+          <base href="https://electron.stp-prod.collegeboard.org/"> 
+  <script>
+window.ipcRenderer = {
+  send: (...args) => console.log('[send]', args),
+  on: (...args) => {
+    console.log('[on]', args)
+    if (args[0] === 'update:not-available') args[1]()
+  },
+  removeListener: (...args) => console.log('[removeListener]', args),
+  invoke: (...args) => (console.log('[invoke]', args), "{}"),
+} </script>
+        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
```

Comment:
This file will be deployed so I should probably neutralize this link (the funniest way would probably be to make it an http link)